cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -20,6 +20,13 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 * You can now use {CNVProductName} with either the xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#learn-about-ovn-kubernetes[OVN-Kubernetes] or the xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] network provider.
 
+// CNV-3881 Release note to announce SVVP Certification
```

Comment:
@bgaydosrh Can you please move this to replace the placeholder(`// CNV-4310/CNV-3881 SVVP`)  at L48

Reply:
Will do today, @aburdenthehand. Thanks. Bob

Reply:
Done.